templates/partial/_careers_navigation.html

```diff
@@ -0,0 +1,334 @@
+<div class="dropdown-window-overlay fade-animation"></div>
+<div class="dropdown-window slide-animation">
+	<div class="u-hide p-section" id="careers-menu" aria-hidden="true">
+		<div class="u-fixed-width p-strip">
+			<h2><a class="p-link is-dark" href="/careers">Careers <br class="u-hide--medium u-hide--large" />overview</a></h2>
+		</div>
+		<div class="row">
+			<div class="col-9 col-medium-4" id="careers-navigation-container">
+				<div class="u-sv2">
+					<h4 class="p-muted-heading p-text--small-caps muted-color">Roles by department</h4>
+				</div>
+				<hr class="p-rule is-dark is-bold">
+				<div class="row">
+					<div class="col-2 col-medium-2 col-small-12 p-block">
```

Comment:
`p-block` doesn't do anything here

Reply:
It adds spacing on the bottom for smaller screens